azure-functions/acbs-function/src/functions/acbs-amend-facility-loan-record.js

```diff
@@ -38,11 +38,22 @@ df.app.orchestration('acbs-amend-facility-loan-record', function* amendFacilityL
 
     const { facilityId, facility, amendments, fmr } = payload;
     const { amendment } = amendments;
+
     let facilityLoanRecordAmendments;
 
     // 1.1. Facility Loan Record (FLR) amendment mapping
     const flrMApped = mappings.facility.facilityLoanAmend(amendments, facility, fmr);
 
+    /**
+     * When facility amendment is applied on facility type `Loan`,
+     * with only `amount` attribute execution should be terminated.
+     */
+    if (!flrMApped?.length) {
```

Comment:
Unless `flrMApped` is intentional, whilst we're here, can we rename to `flrMApped` to `flrMapped`?

Reply:
Ah apologies, good spot.

Reply:
Bump @abhi-markan